datalad/tests/test_cmdline_main.py

```diff
@@ -66,8 +66,7 @@ def test_version():
 
 
 def test_help_np():
-    with patch.dict('os.environ', {'DATALAD_HELP2MAN': '1'}):
```

Comment:
rright, was removed in aeeee9fcf26b351f1621e0fa94981503c82a189d . So good to proceed with this one.  I will test and merge as is, but then still introduce MANPATH env var handling so we are all kosher on that regard